dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHV2IT.java

```diff
@@ -168,4 +182,21 @@ public void toolsIdGet4Workflows() throws Exception {
         responseObject = response.readEntity(ToolV2.class);
         assertThat(MAPPER.writeValueAsString(responseObject)).contains("author4");
     }
+
+    @Test
+    public void cwlrunner() throws Exception {
+        CommonTestUtilities.setupSamePathsTest(SUPPORT);
+        ProcessBuilder pb = new ProcessBuilder("cwl-runner",
```

Comment:
We already have existing code for executing commands, use that https://github.com/ga4gh/dockstore/blob/develop/dockstore-common/src/main/java/io/dockstore/common/Utilities.java#L96 since it avoids common pitfalls with ProcessBuilder